models/m_payment_mollie_api.erl

```diff
@@ -195,40 +221,42 @@ fetch_payments_since(Oldest, NextLink, Acc, Context) ->
             Error
     end.
 
-handle_missed_recurrent(#{ <<"recurringType">> := <<"recurring">> } = JSON, Context) ->
+handle_missed_recurrent(#{ <<"sequenceType">> := <<"recurring">> } = JSON, Context) ->
     #{
         <<"id">> := ExtId,
-        <<"links">> := #{
-            <<"webhookUrl">> := WebhookUrl
-        }
+        <<"webhookUrl">> := WebhookUrl
```

Comment:
Vaag dat het hier niet met `_links` en `href` waarde is, of is dat wel logisch?

Reply:
Indeed, a bit inconsistent in their API.

https://docs.mollie.com/reference/get-payment

Reply:
I guess `webhookUrl` is something we supplied, and because of that a property.
Whereas the `_links` are generated by Mollie.